app/src/main/java/org/mozilla/focus/utils/ViewUtils.java

```diff
@@ -26,6 +26,13 @@
 
 public class ViewUtils {
     /**
+     * Flag of imeOptions: used to request that the IME does not update any personalized data such
+     * as typing history and personalized language model based on what the user typed on this text
+     * editing object.
+     */
+    public static final int IME_FLAG_NO_PERSONALIZED_LEARNING = 0x01000000;
```

Comment:
This constant is directly available in API 26, it may be a good idea to use it after #1194 gets merged.